### PR TITLE
New section

### PR DESCRIPTION
**Pull request by prashant280920**
i have added three more members in other programs section of Gsoc.

### CODE REVIEW

File: gsoc/index.html [hunk 1]
```diff
@@ -581,6 +581,37 @@ <h4><b>Yash Varshney</b></h4>
...
+               <h4><b>Nidhi Jain</b></h4>
+               <!--<a href="#" class="icon icon-twitter"></a>-->
+               <a href="https://www.facebook.com/profile.php?id=100026427400017" class="fab fa-facebook-f social-button"></a>
+               <a href="https://github.com/nidhijainpnp" class="social-button fab fa-github"></a>
+               <p>LINUX FOUNDATION CommunityBridge 2020 - Open Printing</p>
+            </div>
+            <div class="col-lg-4 centered">
+               <img class="img img-circle" src="img/people/jhanvigupta.jpg" height="120px" width="120px" alt="Piyush Goyal">
+               <br>
+               <h4><b>Jhanvi Gupta</b></h4>
```
Comment (vsvipul) [inline gsoc/index.html]: spelling

Comment (vsvipul) [reply gsoc/index.html]: change for image file as well.

File: gsoc/index.html [hunk 2]
```diff
@@ -581,6 +581,37 @@ <h4><b>Yash Varshney</b></h4>
...
+               <!--<a href="#" class="icon icon-twitter"></a>-->
+               <a href="https://www.facebook.com/jahnvi.gupta.3956" class="fab fa-facebook-f social-button"></a>
+               <a href="https://github.com/jahnvigupta" class="social-button fab fa-github"></a>
+               <p>LINUX FOUNDATION CommunityBridge 2020 - Moja Global</p>
+            </div>
+            <div class="col-lg-4 centered">
+               <img class="img img-circle" src="img/people/dipanshuverma.jpg" height="120px" width="120px" alt="Yash Varshney">
```
Comment (vsvipul) [inline gsoc/index.html]: why alt yash varshney ? change alts for others as well.

**Comment by prashant280920** [comment]
Ok sir

**Comment by vsvipul** [comment]
@prashant280920 Add Piyush Goyal for Google Season Of Docs 2020 as well.

**Comment by vsvipul** [comment]
@kartik1000  review this when free. make changes if required and merge.

**Comment by kartik1000** [comment]
@vsvipul changes look ready to be merged. Merging

**Pull request merged**